### DISCUSSION TITLE
react-bootstrap-confirmation

### DISCUSSION THREAD

**Discussion by SonGokussj4** [DefinitelyTyped/DefinitelyTyped#61177] [Request a new @types package] [answered]
Hello, is it possible to add `react-bootstrap-confirmation` please?

[https://www.npmjs.com/package/react-bootstrap-confirmation](https://www.npmjs.com/package/react-bootstrap-confirmation)

**Comment by jbovone** [comment, marked as answer]
I will take it, should be quick. :)

**Comment by jbovone** [comment]
Hi! @SonGokussj4

There is a draft here:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61216

You can review it if you like.

**Reply by SonGokussj4** [reply]
So I tried to install it and it got stuck for 10 minutes now, not sure what to do
I don't have much experience with `npm` and only a few days experience with `typescript`
```bash
$ npm i --save-dev DefinitelyTyped/DefinitelyTyped#pull/61216 --legacy-peer-deps
(#################⠂) ⠴ reify:@next/swc-android-arm-eabi: timing reifyNode:node_modules/@next/swc-android-arm64 Comp
```

Edit: Spoke too soon. After 17 minutes it finished installing.

**Reply by SonGokussj4** [reply]
It still lights as red. Not sure why. 

![image](https://user-images.githubusercontent.com/13331858/178695314-fd8c45be-3d19-4542-a401-edf9ac2b8973.png)

In packages, I have:

```
#package.json
...
"devDependencies": {
    "@types/node": "^18.0.3",
    "@types/react": "^18.0.9",
    "@types/react-bootstrap-confirmation": "github:DefinitelyTyped/DefinitelyTyped#pull/61216",
    "eslint": "8.12.0",
    "eslint-config-next": "12.1.4",
    "local-ssl-proxy": "^1.3.0",
    "process": "0.11.10",
    "typescript": "^4.7.2"
  }
```

**Comment by jbovone** [comment]
Did you manually checked that the package was in the node_modules/@types? I don't sure that it will work doing it that way, because DT-Tools does stuff with the source types. 

It's already merged, and seems to be working fine anyways. If you find something wrong you can do an issue.

**Reply by SonGokussj4** [reply]
I did not find it in my `node_modules/@types`, but I was able to uninstall it and install from official `npm install` and it works! Thank you very much.
